<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointItem</class>
 <widget class="QWidget" name="PointItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPointItem</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_PointItem">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelP">
          <property name="text">
           <string>P0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditX"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditY"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
